### ISSUE TITLE
Decreased level of coenzyme Q10 in skeletal muscle

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7680]
**Preferred term label:**
Decreased level of coenzyme Q10 in skeletal muscle

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
607426 
PMID: 16400613  1/1
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
